pkg/controller/import-controller.go

```diff
@@ -247,6 +285,7 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 			log.V(1).Info("Pod requires scratch space, terminating pod, and restarting with scratch space", "pod.Name", pod.Name)
 			scratchExitCode = true
 			anno[AnnRequiresScratch] = "true"
+			anno[AnnScratchPvc] = scratchNameFromPvc(pvc)
```

Comment:
This one really needed? Looks like we create it above when we determine that we need scratch space.

Reply:
see my last comment on storing names in annotations

Reply:
actually not, we create scratch name once, and then it is always stored in a Volume definition on the pod

Reply:
so I have removed this annotation, as it is not needed, and change to read from pod spec, if available